apps/web/app/(app)/stats/LargestEmails.tsx

```diff
@@ -36,6 +37,16 @@ export function LargestEmails(props: { refreshInterval: number }) {
 
   const { expanded, extra } = useExpanded();
 
+  // Function to calculte total attachement size
+  const calculateTotalAttachmentSize = (attachments?: Attachment[]): number => {
+    return (
+      attachments?.reduce(
+        (sum: number, attachment: Attachment) => sum + attachment.size,
+        0,
+      ) || 0
+    );
+  };
+
```

Comment:
Maybe use `sum` from lodash just because we use this in other places in the app anyway. And just easier to read a one liner over `reduce`.

Reply:
Done!